openbb_platform/extensions/platform_api/pyproject.toml

```diff
@@ -15,7 +15,7 @@ openbb-api = "openbb_platform_api.main:main"
 
 [tool.poetry.dependencies]
 python = ">=3.9.21,<3.13"
-openbb-core = "^1.4.3"
+openbb-core = "^1.4.6"
```

Comment:
Something I wanted to ask. Would it at some point be possible to make this package not rely on openbb-core?

Rationale:
I would like to use this package as a tool to work on FastAPI based app backends. Specifically to declare widgets at endpoint level and generate the widgets.json programmatically. I don't require pulling all the dependencies of openbb-core to those project. The --no-deps is an option but it won't work when specifying the dependencies in a pyproject.toml.
Heavy coupling with other openbb extensions and the core is what's "blocking" me from more adoption of this as a pure tool for parsing openapi schemas to integrate with the openbb workspace.

Reply:
Not sure I understand the request here.

What heavy coupling are you talking about?

`openbb-core` is not a heavy package and it gives you the base packages like FastAPI and Pydantic - you already need these packages. We need `openbb-core` for the FastAPI instance.

`openbb-core` is also responsible for `.env`, `user_settings.json` and `system_settings.json`.